tests/ignite/distributed/utils/test_native.py

```diff
@@ -37,7 +39,6 @@ def _test_native_distrib_single_node_launch_tool(backend, device, local_rank, wo
 @pytest.mark.skipif(not has_native_dist_support, reason="Skip if no native dist support")
 @pytest.mark.parametrize("init_method", [None, "tcp://0.0.0.0:22334", "FILE"])
 def test_native_distrib_single_node_launch_tool_gloo(init_method, get_fixed_dirname, local_rank, world_size):
-
```

Comment:
Are you using black or some other auto code formatter ?
We may also first upgrade black version first before this PR such that we do not mix code formatting and new feature.

Reply:
yes, black is updated now

Reply:
Would you mind to send a draft PR with updated code and our pinned ufmt, black etc version ?

Reply:
Versions:
>>> black.__version__
'21.12b0'

usort, version 1.0.5

ufmt, version 2.0.1

>>> flake8.__version__
'6.0.0'

mypy 1.2.0 (compiled: yes)